docs/participate/translate.md

```diff
@@ -37,6 +37,10 @@ As Nakamoto envisioned a more equitable and decentralized future, Subspace Labs
 
 <TranslationLeaderboard />
 
+:::note
+Please note that contributions made on the last day of the month may not be counted in the leaderboard!
```

Comment:
Any evidence of this? I assume this is related to the [translation contest](https://forum.autonomys.xyz/t/translathon-enhance-our-crowdin-translations-and-win-50/3453)? I think it would make more sense to add a note to the forum post that the leaderboard snapshot is taken at 00:00UTC on the last day of the month.

@EmilFattakhov is that accurate?

@ryyeung can we add something to the forum post please?

Reply:
I see where @RomanLabGit is coming from based on the previous conversations in Discord, but it's inaccurate. I believe we should change the note about the time of the snapshot taken - 00:00 UTC time as you suggested @jim-counter